README.md

```diff
@@ -1,24 +1,204 @@
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><p> <b>Ruby: </b> The primary programming language for the project.</p></li>
+    <li><p><b>Rails: </b> It is a ruby frameworks.</p></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>none</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+## Key Features <a name="key-features"></a>
+
+- Users functionality
+- Comments section
+- Like section
+- Post section
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A Computer (MAC or PC)
+- Code editor (VSCode,Atom etc...)
+- A browser (Chrome,Mozilla,Safari etc...)
+- Version Control System (Git and Github)
+- Rails installed: To run a Rails project, users must have Rails installed on their system. You can include the minimum required version of Rails needed to run the project.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:david30-maker/rails-blog-app.git
+```
+
+## Install
+
+Install this project with:
+
+- Install RailsInstaller [Win](https://railsinstaller.org/)
+- Install Rails for [macOS](https://www.rails-lang.org/en/documentation/installation/#homebrew)
+- Install Rails for [Linux](https://www.makeuseof.com/install-rails-on-linux/)
+
+## Usage
+
+To run a Rails file in the command line, follow these steps:
+
+Open a command prompt or terminal window on your computer.
+
+Navigate to the root directory where the Rails file is stored using the cd command.
+
+Type the following command to run the Ruby file:
+
+```sh
+rails server
+```
+
+### - where filename.rb is the name of the Railsfile you want to run.
+
+## Run tests
+
+```sh
+rspec spec/filename.rb
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Okpah Igbo**
+
+- GitHub: [@david30-maker](https://github.com/david30-maker)
+- Twitter: [@davidigbo1](https://twitter.com/davidigbo1)
+- LinkedIn: [davidigbo/](https://www.linkedin.com/in/davidigbo/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ]   Processing data
+- [ ] Add controller and view components
+- [ ] Add authorization
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://git@github.com:david30-maker/rails-blog-app.git/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] The issues page link is broken, Please make sure to fix this. It will make the README file more professional.